js-miniapp-bridge/src/types/fetch.ts

```diff
@@ -0,0 +1,4 @@
+export type MiniAppRequestInit = Pick<
```

Comment:
How about the name `FetchRequest`? I don't think we need to add the `MiniApp` prefix to our types because the user will be able rename them when importing, if needed: i.e. `import {WebRequest as MiniAppRequest} from 'js-miniapp-sdk'`